cpp/ql/src/Likely Bugs/Likely Typos/ExprHasNoEffect.ql

```diff
@@ -84,6 +84,7 @@ where
   not peivc.getEnclosingFunction().isDefaulted() and
   not exists(Macro m | peivc = m.getAnInvocation().getAnExpandedElement()) and
   not peivc.isFromTemplateInstantiation(_) and
+  not peivc.isFromUninstantiatedTemplate(_) and
```

Comment:
I'm a bit surprised we didn't already have this exception, given the number of other exceptions!